### PR TITLE
Download Tokens & Concordances place of Generate Tokens

### PR DESCRIPTION
**Pull request by HimanshuBCS**
I have added the Download Tokens & Concordances in place of Generate Tokens. After click on  Download Tokens & Concordances button able to see concordances with corresponding tokens.
PFA xls attached as image.
tokens with concordances

### CODE REVIEW

**Comment by HimanshuBCS** [comment]
I have completed UI for the task to generate concordance for the particular tokens in UI. 
Also find attached screenshot for the same.
ui

**Comment by HimanshuBCS** [comment]
Ready for the review.

File: src/GetConcordances.js [hunk 1]
```diff
@@ -53,6 +53,7 @@ class GetConcordances extends Component {
 
 //To get concordances for a particular token
   getConcordances(e){
+    console.log("hello")
```
Comment (udkumar) [inline src/GetConcordances.js]: Remove the console.

Comment (HimanshuBCS) [inline src/GetConcordances.js]: Done.

File: src/BookName.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+const booksName = [
+  'GEN', 'EXO', 'LEV', 'NUM', 'DEU', 'JOS', 'JDG', 'RUT', '1SA', '2SA', '1KI', '2KI', '1CH', '2CH', 'EZR', 'NEH', 'EST', 'JOB',
+  'PSA', 'PRO', 'ECC', 'SNG', 'ISA', 'JER', 'LAM', 'EZK', 'DAN', 'HOS', 'JOL', 'AMO', 'OBA', 'JON', 'MIC', 'NAM', 'HAB', 'ZEP',
+  'HAG', 'ZEC', 'MAL', 'MAT', 'MRK', 'LUK', 'JHN', 'ACT', 'ROM', '1CO', '2CO', 'GAL', 'EPH', 'PHP', 'COL', '1TH', '2TH', '1TI',
+  '2TI', 'TIT', 'PHM', 'HEB', 'JAS', '1PE', '2PE', '1JN', '2JN', '3JN', 'JUD', 'REV'
+];
+
+class BookName extends Component {
+  componentWillMount = () => {
+    console.log(new Set())
```
Comment (udkumar) [inline src/BookName.js]: Why is console here?

File: src/BookName.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    } else {
+      this.selectedCheckboxes.add(label);
+    }
+  }
+
+  handleFormSubmit = formSubmitEvent => {
+    formSubmitEvent.preventDefault();
+
+    for (const checkbox of this.selectedCheckboxes) {
+      console.log(this.selectedCheckboxes);
```
Comment (udkumar) [inline src/BookName.js]: Why is console here?

File: src/BookName.js (hunk 2 above)
Comment (HimanshuBCS) [inline src/BookName.js]: I have removed the console.

File: src/BookName.js (hunk 3 above)
Comment (HimanshuBCS) [inline src/BookName.js]: I have removed the console.

**Pull request merged**